pkg/kubectl/cmd/apply.go

```diff
@@ -220,7 +230,13 @@ func RunApply(f cmdutil.Factory, cmd *cobra.Command, out io.Writer, options *App
 			helper := resource.NewHelper(info.Client, info.Mapping)
 			patcher := NewPatcher(encoder, decoder, info.Mapping, helper, overwrite)
 
-			patchBytes, err := patcher.patch(info.Object, modified, info.Source, info.Namespace, info.Name)
+			infos := []*resource.Info{info}
```

Comment:
`info.VersionedObject` to my surprise is of type `interface {}` and might be `nil` as well, so I'd have to cast it to `runtime.Object` and if that fails fall back to `resource.AsVersionedObject`. Or stay with current solution? Which one you prefer?